src/components/Selector.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import uuid from "react-uuid";
+
+function Select({ value, onChange, options }) {
+  return (
+    <select onChange={onChange} value={value}>
+      {options.map((element, index) => (
+        <option key={uuid()} label={options[index].label}>
+          {options[index].value}
+        </option>
+      ))}
+    </select>
```

Comment:
Use the element directly instead of relying on index in `.map` 😄 + added restructuring to map
```suggestion
    <select onChange={onChange} value={value}>
      {options.map((({ value, label })) => (
        <option key={value} label={label}>
          {value}
        </option>
      ))}
    </select>
```

Reply:
done, I had to change "function Select({ selectedValue, onChange, options })" from value to selectedValue, because it was declared twice